index.html

```diff
@@ -0,0 +1,363 @@
+                avoided.
+              </p>
+              <p>
+                Not properly updating your systems makes you an easy to reach
+                target and hackers will often go after the lowest hanging fruit.
+                When it comes to maintaining a strong cybersecurity network the
+                best thing a user can do is to prevent problems before they
+                occur, and the easiest way to do that is by making sure that all
+                systems are updated and updated regularly.
+              </p>
+            </div>
+            <div class="bottom-info">
+              <img src="images/Letter_i.png" alt="#" />
+              <span>Based on the 2018 Verizon Data Breach Report</span>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="accord-container line">
+        <button class="head-accordion">
+          <span class="number">4</span>
+          <span class="accord-text">Secure Credentials</span>
+          <span class="accordion-button"></span>
+        </button>
+        <div class="panel">
+          <img src="images/groop4.png" alt="#" class="main-icon" />
+          <div class="footer-accordion-container">
+            <div class="footer-accordion">
+              <p>
+                There’s nothing hackers love more than a dismissed update
+                reminder. We keep our team’s computers in shape and up to date
+                so that they aren’t subject to attacks that could otherwise be
+                avoided.
+              </p>
+              <p>
+                Not properly updating your systems makes you an easy to reach
+                target and hackers will often go after the lowest hanging fruit.
+                When it comes to maintaining a strong cybersecurity network the
+                best thing a user can do is to prevent problems before they
+                occur, and the easiest way to do that is by making sure that all
+                systems are updated and updated regularly.
+              </p>
+            </div>
+            <div class="bottomIcon">
+              <img src="images/Letter_i.png" alt="#" />
+              <span>Based on the 2018 Verizon Data Breach Report</span>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="glossary-contaner">
+        <img src="images/skout-glossary.svg" id="head-glossary" alt="#" />
+      </div>
+    </div>
+
+    <section class="section">
+      <div class="section-wrapper max-width">
+        <div class="slider-container">
+          <div class="slider-head"></div>
+          <div class="slider-button-container">
+            <button class="slider-button" value="right"></button>
+            <button class="slider-button rotate-arrow" value="left"></button>
+          </div>
+        </div>
+        <div class="slider-text-container">
+          <div>
+            <div>
+              <p class="slider-title">Ransomware</p>
```

Comment:
if class name contain word `title`, why do you use tag `p` instead of heading tag?

Reply:
renemed